lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -104,6 +104,13 @@
       - Force update even if no changes require it.
       - Corresponds to the C(--force) option of C(docker service update).
       - Requires API version >= 1.25.
+  groups:
+    type: list
+    description:
+      - List of additional group names and/or IDs that the container process will run as.
+      - Corresponds to the C(--group) option of C(docker service update).      
```

Comment:
There are trailing whitespaces in this line and the following two (that's why tests are failing).

Reply:
Thanks, somehow I always miss those.

Reply:
NVM, I ran sanity checks on `docker_container` instead of `docker_swarm_service`..